### DISCUSSION TITLE
You should look this

### DISCUSSION THREAD

**Discussion by Sreehari521** [harso-css/harso-css#4] [General]
@e-coders You made the official website nicely, but it SHOULD BE NOT HERE. It should be in https://github.com/harso-css/harso-css.github.io

**Comment by devashish2024** [comment]
No, It is documenation. may be it in a different repo? If the original files are lost, Click on __commits in main repo page and then choose a commit that is came before merging, and then get files again.

**Reply by Sreehari521** [reply]
It's not lost, It is in harso-css.github.io

**Comment by devashish2024** [comment]
ok
